MdePkg/MdePkg.dec

```diff
@@ -2433,6 +2433,34 @@
   # @Prompt Time-out for a response, internal
   gEfiMdePkgTokenSpaceGuid.PcdIpmiSsifResponseRetryIntervalMicrosecond|60000|UINT32|0x00000036
 
+  ## This is requester's Software IDs for IPMI Serial.
+  #  This byte is 20h when the BMC is the requester (section 14.4.3).
+  # @Prompt IPMI Serial requester Software ID
+  gEfiMdePkgTokenSpaceGuid.PcdIpmiSerialRequesterAddress|0x20|UINT8|0x00000050
+
+  ## This is responder's Software IDs for IPMI Serial.
+  #  This byte is 20h when the BMC is the responder (section 14.4.3).
+  # @Prompt IPMI Serial responder Software ID
+  gEfiMdePkgTokenSpaceGuid.PcdIpmiSerialResponderAddress|0x20|UINT8|0x00000051
+
+  ## This is requester's LUN for IPMI Serial.
+  # @Prompt IPMI Serial requester LUN
+  gEfiMdePkgTokenSpaceGuid.PcdIpmiSerialRequesterLUN|0x0|UINT8|0x00000052
+
+  ## This is responder's LUN for IPMI Serial.
+  # @Prompt IPMI Serial responder LUN
+  gEfiMdePkgTokenSpaceGuid.PcdIpmiSerialResponderLUN|0x0|UINT8|0x00000053

```

Comment:
Should be PcdIpmiSerialResponderLun to follow naming conventions.

Reply:
Acked.